### PR TITLE
Christmasbranch

### PR DESCRIPTION
**Pull request by vs77bb**
##### Description

Made some changes to automated emails 
- no more asking for physical mailing addresses, we'll send shirts online
- fixed some post-bounty retention email copy 

##### Refers/Fixes
changed the email to send after 60 days of inactivity as it current reads 'every few months'... and i think thats less spammy
 
image

##### Testing

would love to know if there are any tests i should run for email updates, might have some more fun here in the new year!

### CODE REVIEW

**Comment by danlipert** [comment]
@vs77bb nice! looks like there are some newsletter changes as well? Or is that an old newsletter... I can't remember. Anyway, let me know if you meant to make the newsletter changes and if not I'll show you how to fix this PR

File: app/retail/emails.py [hunk 1]
```diff
@@ -443,7 +443,7 @@ def render_admin_contact_funder(bounty, text, from_user):
     return response_html, response_txt
 
 
-def render_funder_stale(github_username, days=30, time_as_str='about a month'):
+def render_funder_stale(github_username, days=60, time_as_str='a couple months'):
```
Comment (owocki) [inline app/retail/emails.py]: will need to update this line in the crontab too.  this change only edits the default arguments, not the actually executed ones

```
15 10 * * * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash funder_stale_email 30  >> /var/log/gitcoin/funder_stale_email.log  2>&1
```

Comment (thelostone-mc) [reply app/retail/emails.py]: taken care

**Comment by thelostone-mc** [comment]
raised a PR removing the newsletter commit over at https://github.com/gitcoinco/web/pull/5699
@vs77bb if that looks alright we can get that in ^_^

**Pull request closed without merging**